pkgs/development/tools/database/sqlitebrowser/default.nix

```diff
@@ -1,38 +1,29 @@
 { mkDerivation, lib, fetchFromGitHub, cmake, antlr
-, qtbase, qttools, qscintilla, sqlite }:
+, qtbase, qttools, sqlite }:
 
 mkDerivation rec {
-  version = "3.10.1";
-  name = "sqlitebrowser-${version}";
+  version = "3.11.2";
+  pname = "sqlitebrowser";
 
   src = fetchFromGitHub {
-    repo   = "sqlitebrowser";
-    owner  = "sqlitebrowser";
+    repo   = pname;
+    owner  = pname;
     rev    = "v${version}";
-    sha256 = "1brzam8yv6sbdmbqsp7vglhd6wlx49g2ap8llr271zrkld4k3kar";
+    sha256 = "0ydd5fg76d5d23byac1f7f8mzx3brmd0cnnkd58qpmlzi7p9hcvx";
   };
 
-  buildInputs = [ qtbase qscintilla sqlite ];
+  buildInputs = [ qtbase sqlite ];
 
   nativeBuildInputs = [ cmake antlr qttools ];
 
+  cmakeFlags = [ "-DFORCE_INTERNAL_QSCINTILLA=ON" ];
```

Comment:
you've explained this in the message, but please do so here as well

Reply:
fixed :)